indico/web/client/js/react/components/principals/PrincipalListField.jsx

```diff
@@ -100,6 +102,7 @@ const PrincipalListField = (props) => {
                                        name={data.name}
                                        detail={data.detail}
                                        isGroup={data.group}
+                                       isPendingUser={!data.group && data.userId === null}
```

Comment:
Isn't it enough to check if `data.userId === null`?

Reply:
Yes, but that's just because `null !== undefined` and I really don't want to depend on that :)